pdf/creator/table.go

```diff
@@ -208,7 +210,50 @@ func (table *Table) GeneratePageBlocks(ctx DrawContext) ([]*Block, DrawContext,
 				// Add diff to last row
```

Comment:
Should end with period as godoc style.

Reply:
not so important within function (does not go into godoc), but sure will fix as this is a sentence and is more consistent.